lib/rollbax/notifier.ex

```diff
@@ -44,4 +44,10 @@ defmodule Rollbax.Notifier do
     %{level: Keyword.get(config, :level, :error),
       metadata: Keyword.get(config, :metadata, [])}
   end
+
+  defp ts_to_unix({{_yy, _mm, _dd} = date, {h, m, s, _ms}}) do
```

Comment:
@lexmag I used `{hour, min, sec, _millisec}` here, I'm not a fan of just `_` for ms. Wdyt?